.github/workflows/test.yml

```diff
@@ -36,4 +36,9 @@ jobs:
         pip install .[ml]
     - name: Test with pytest
       run: |
-        pytest -m pytest --random-order --random-order-bucket=global -m "not api_validation" tests/
+        python -m pytest --random-order --random-order-bucket=global --ignore=docs -m "not api_validation" tests/
+    - name: Analysing the code with pylint
```

Comment:
Maybe it's better to add a new workflow for the pylint rather than reuse test?
I think motivation might be:

- Tests and pylint can fail separately. In case of they joined into a single task it's not immediately obvious which one did fail
- I suppose there are no need to test pylint on a 3 different OS

Reply:
+